app/views/show.erb

```diff
@@ -1,29 +1,29 @@
 <%# TODO: add reveal this button prior showing bin %>
 <div id="decrypt-header">
-<h1> Someone shares a secret with you </h1>
+<h1><%= t :show_header %></h1>
 </div>
 
 <div id="bin-content">
-  <h1>Your decrypted message:</h1>
+  <h1><%= t :decrypted_header %>:</h1>
   <span id="bin-id" class="hidden"><%= @bin.id %></span>
   <textarea id="dec-msg" readonly rows="20" cols="40"></textarea>
-  <p>This message was deleted from the server and can't be shown twice</p>
+  <p><%= t :deleted_info %></p>
 </div>
 
 <div id="reveal-content">
 <% if @bin.has_password %>
   <div id="pw-enter">
   <p>
-    The sender has set an additional password that allows to access the secret.
+    <%= t :additional_password_info %>
   </p>
   <div id="pwenterset">
     <input id="passwd" type="text" name="passwd" class="textinput" placeholder="Enter password">
```

Comment:
"Enter password" placeholder needs to be replaced too

Reply:
fixed